src/index.js

```diff
@@ -1,11 +1,54 @@
+import util from 'util';
 import app from './app';
-import config from './config/config.json';
-import { echoServer, htmlServer, jsonServer, plainTextServer, logger } from './middlewares';
+import { CityController, CredentialsController, ProductController, ReviewController, UserController } from './controllers';
+import { ErrorHandler } from './helpers';
+import initLocale from './lang';
+import { cityServer, echoServer, htmlServer, jsonServer, plainTextServer, initPassport, connectors, logger } from './middlewares';
 
 const port = process.env.PORT || 8090;
-app.listen(port, () => logger.info(`App ${config.server_listening} ${port}`));
+
+initLocale();
+initPassport();
+
+(async () => {
+  try {
+    await connectors.MONGO.connect();
+
+    CityController.addCity('Brest', 'Belarus', false, 52.097621, 23.734050);
```

Comment:
Pretty nice try but when you run it the 2nd time you add the same data again (+ there are `E11000 duplicate key error collection` errors).
I'd move it to separate file and add to npm scripts.